pkg/products/marin3r/rateLimitService.go

```diff
@@ -381,3 +402,119 @@ func GetRateLimitFromConfig(c *corev1.ConfigMap) (*yamlRateLimit, error) {
 	}
 	return ratelimitconfig.Descriptors[0].RateLimit, nil
 }
+
+func (r *RateLimitServiceReconciler) getConfigMapConfiguration(ctx context.Context, client k8sclient.Client, installation *integreatlyv1alpha1.RHMI) (*yamlRoot, error) {
```

Comment:
Good call moving it to a function, but I would break this function into two `getConfigMapConfiguration` and `getConfigMapConfigurationPerTenant` to avoid the if else statement in the code or just use the if statement to include the tenant descriptor to the array of descriptor.

Reply:
Thanks Jose! Updated.